File: sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/LineLength.vb
Imports System
Imports System.Collections.Generic

Namespace Tests.Diagnostics

    Public Class LineLength
        Public Sub LineLength()

            Console.WriteLine("This is OK...") ' With these comments...........................................................
            Console.WriteLine() ' Noncompliant   {{Split this 128 characters long line (which is greater than 127 authorized).}}
        End Sub
    End Class
End Namespace

File: sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/LineLength.cs
using System;
using System.Collections.Generic;

namespace Tests.Diagnostics
{
    public class LineLength
    {
        public LineLength()
        {
            Console.WriteLine("This is OK..."); // with these comments.........................................................
            Console.WriteLine(); // Noncompliant {{Split this 128 characters long line (which is greater than 127 authorized).}}
        }
    }
}
